lib/wordnet/synset.rb

```diff
@@ -100,13 +198,18 @@ def hypernym
       relation(HYPERNYM)[0]
     end
 
+    # Get the parent synset (higher-level category, i.e. fruit -> reproductive_structure).
+    def hypernyms
+      relation(HYPERNYM)
+    end
+
     # Get the child synset(s) (i.e., lower-level categories, i.e. fruit -> edible_fruit)
-    def hyponym
+    def hyponyms
       relation(HYPONYM)
     end
 
     # Get the entire hypernym tree (from this synset all the way up to +entity+) as an array.
-    def expanded_hypernym
+    def expanded_first_hypernyms
```

Comment:
I don't think I understand what this method does, or how it differs from `expanded_hypernym(s)`.